### PR TITLE
Set _use_sdpa to False to disable torch sdpa

### PR DESCRIPTION
**Pull request by hlahkar**
Fixes accuracy for Llama fine-tuning & LLama inference seen with PyTorch 2.2 builds

### CODE REVIEW

**Review by vivekgoe — APPROVED** [review]

**Comment by hlahkar** [comment]
Corresponding OH PR is https://github.com/huggingface/optimum-habana/pull/753
